src/main/kotlin/defineWorkspace.kt

```diff
@@ -3,58 +3,70 @@ package uk.gov.justice.hmpps.architecture
 import com.structurizr.Workspace
 import com.structurizr.model.CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy
 import com.structurizr.model.Enterprise
+import com.structurizr.model.Model
+import com.structurizr.view.ViewSet
 
-fun defineWorkspace(): Workspace {
-  val enterprise = Enterprise("HM Prison and Probation Service")
-  val workspace = Workspace(enterprise.name, "Systems related to the custody and probation of offenders")
-  workspace.id = 56937
-  workspace.model.enterprise = enterprise
+private val ModelItems = listOf(
```

Comment:
Is `ModelItems` the right casing? It makes this look like a class, but I wonder if it should be `modelItems` or `MODEL_ITEMS` if it's a const? But that could just be me being a Kotlin novice too...

Reply:
You're right! Fixed in e4c0401c59f22f0c1905d853cb021b444f94752f